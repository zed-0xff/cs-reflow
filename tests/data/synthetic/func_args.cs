void case1(int x) {
    if (x >= 0)
        maybe_called();
}

void case2(uint x) {
    if (x >= 0)
        always_called();
}
