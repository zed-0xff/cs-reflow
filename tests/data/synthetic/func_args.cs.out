void case1(int x)
{
    if (x >= 0)
    {
        maybe_called();
    }
}

void case2(uint x)
{
    always_called();
}
